src/views/Give/Give.tsx

```diff
@@ -23,6 +23,13 @@ function a11yProps(index: number) {
   };
 }
 
+export const isSupportedChain = (chainID: NetworkId): boolean => {
```

Comment:
It doesn't make sense for this helper function to be in a view. Consider shifting it to another file.